db/migrations/20230628144927_mission_reward_migration.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class MissionRewardMigration extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up(): void
+    {
+        // Modify missions table
+        $this->execute("
+            ALTER TABLE `missions` 
+                ADD `rewards` VARCHAR(500) NOT NULL DEFAULT '[]';
```

Comment:
```suggestion
                ADD `rewards` VARCHAR(2000) NOT NULL DEFAULT '[]';
```
Lean really on the side of "way larger than we could ever fathom needing" with these if not using `TEXT`, it's tricky to track down when they're getting truncated